packages/syntax-core/src/Popover/Popover.stories.tsx

```diff
@@ -0,0 +1,310 @@
+import { useState, useRef, useEffect, type ReactElement } from "react";
+import type { StoryObj, Meta } from "@storybook/react";
+import Box from "../Box/Box";
+import Typography from "../Typography/Typography";
+import Button from "../Button/Button";
+import Popover from "./Popover";
+import IconButton from "../../../syntax-core/src/IconButton/IconButton";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import Tooltip from "../Tooltip/Tooltip";
+
+const ContentWithTooltips = () => (
```

Comment:
not blocking: something to keep in mind, in places that you're using `ContentWithToolTips` when you click "show code" on storybook, it won't have the expanded code.

so in places you _care_ or _want_ to show code (like in default), i'd probably have this chunk duplicated. but some of the lower stories it probably doesn't matter as much

Reply:
Thank you!  This is the reasoning I was looking for to justify more verbosity in these story files 🙏